src/routes/(console)/project-[region]-[project]/functions/function-[function]/domains/+layout.svelte

```
@@ -0,0 +1,25 @@
+    import { invalidate } from '$app/navigation';
+
+    onMount(() => {
+        return realtime
+            .forProject(page.params.region, page.params.project)
+            .subscribe('console', (response) => {
+                if (
+                    response.events.includes('rules.*.update') ||
+                    response.events.includes('rules.*.create') ||
+                    response.events.includes('rules.*.delete')
+                ) {
+                    if ((response.payload as any)?.deploymentResourceId === page.params.function) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)